### ISSUE TITLE
Suggestion: Turn on linear filtering to improve picture quality

### ISSUE THREAD

**Issue by rsn8887** [p-sam/supermariowar-nx#1]
Thanks for the awesome port! I tried to port 1.8b2 myself, but it always crashed...

To improve picture quality in your port a bit and make the pixels look more "even", but also a bit more blurry, you could do the following:

Before SDL_RenderCopy, here: https://github.com/mmatyas/supermariowar/blob/167ccb716445cd74377962821074a90ab372c065/src/common/gfx/gfxSDL.cpp#L192
, put the instruction
```
SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, "linear");
```

**Comment by rsn8887** [comment]
See my PR 
https://github.com/p-sam/supermariowar/pull/1

**Comment by natinusala** [comment]
Care to post a before / after screenshot of the game ?

**Comment by p-sam** [comment]
Thanks for the PRs, always appreciated. I chose to use the rewrite from mmatyas/supermariowar because it directly supports SDL2, and has network multiplayer. Also, it may even be upstreamed eventually.

I'll answer about the code in the PR directly, but I tried it and as you said it looked blurry. I'll point to you where it's done, if you wanna try yourself, but the original code explicitly hints at nearest already, so I'll leave it as-is and make it togglable in the options.
